doc/source/cluster/running-applications/job-submission/sdk.rst

```diff
@@ -246,4 +247,4 @@ The ``verify`` parameter can be set to override this behavior. For example:
     client = JobSubmissionClient("https://<job-server-url>", verify="/path/to/cert.pem")
 
 will use the certificate found at ``/path/to/cert.pem`` to verify the job server's certificate.
-Certificate verification can be disabled by setting the ``verify`` parameter to ``False``.
```

Comment:
what's the difference between the deleted line and the new line?

Reply:
No idea... Maybe the linter did it?

Reply:
Or a github bug